certbot/cli.py

```diff
@@ -944,6 +946,10 @@ def prepare_and_parse_args(plugins, args, detect_defaults=False):  # pylint: dis
         help=config_help("no_verify_ssl"),
         default=flag_default("no_verify_ssl"))
     helpful.add(
+        ["certificates", "testing"], "--check-ocsp", default="always",
```

Comment:
The `never` option is there in case people don't want `certbot certificates` to be hitting the network.  The `lazy` option claims to be there in case people have a really slow and terrible network, and they don't want to wait for it unnecessarily. But it's not clear that we really need lazy?